Sim/SimDelphesInterface/src/DelphesSimulation.cpp

```diff
@@ -18,37 +43,54 @@ GaudiAlgorithm(name, svcLoc) ,
-   m_jetOutArray        = nullptr;
-   m_metOutArray        = nullptr;
-   m_htOutArray         = nullptr;
-
-   m_eventCounter = 0;
+  m_confReader(nullptr) {
+
+  //declareProperty("filename", m_filename="" , "Name of the HepMC file to read");
+  declareProperty("DelphesCard"      , m_DelphesCard              , "Name of Delphes tcl config file with detector and simulation parameters");
+  declareProperty("HepMCInputFile"   , m_inHepMCFileName          , "Name of HepMC input file; if defined, file read in / if not, data read in directly from the transient data store");
+  declareProperty("ROOTOutputFile"   , m_outRootFileName          , "Name of Delphes Root output file, if defined, the Delphes standard tree write out (in addition to FCC-EDM based output to transient data store)");
+  declareProperty("MuonsOutArray"    , m_DelphesMuonsArrayName    , "Name of Delphes muons array to be written out to FCC-EDM");
+  declareProperty("ElectronsOutArray", m_DelphesElectronsArrayName, "Name of Delphes electrons array to be written out to FCC-EDM");
+  declareProperty("ChargedOutArray"  , m_DelphesChargedArrayName  , "Name of Delphes charged hadrons array to be written out to FCC-EDM");
+  declareProperty("NeutralOutArray"  , m_DelphesNeutralArrayName  , "Name of Delphes neutral hadrons array to be written out to FCC-EDM");
+  declareProperty("PhotonsOutArray"  , m_DelphesPhotonsArrayName  , "Name of Delphes photons array to be written out to FCC-EDM");
+  declareProperty("JetsOutArray"     , m_DelphesJetsArrayName     , "Name of Delphes jets array to be written out to FCC-EDM");
+  declareProperty("METsOutArray"     , m_DelphesMETsArrayName     , "Name of Delphes METs array to be written out to FCC-EDM");
+  declareProperty("SHTsOutArray"     , m_DelphesSHTsArrayName     , "Name of Delphes Scalar HTs array to be written out to FCC-EDM");
+
+  declareInput("hepmc", m_hepmcHandle);
+   
+  declareOutput("genParticles"      , m_handleGenParticles);
+  declareOutput("genVertices"       , m_handleGenVertices);
+  declareOutput("recMuons"          , m_handleRecMuons);
+  declareOutput("recElectrons"      , m_handleRecElectrons);
+  declareOutput("recCharged"        , m_handleRecCharged);
+  declareOutput("recNeutral"        , m_handleRecNeutral);
+  declareOutput("recPhotons"        , m_handleRecPhotons);
+  declareOutput("recJets"           , m_handleRecJets);
+  declareOutput("recMETs"           , m_handleRecMETs);
+
+  declareOutput("recMuonsToMC"      , m_handleRecMuonsToMC);
+  declareOutput("recElectronsToMC"  , m_handleRecElectronsToMC);
+  declareOutput("recChargedToMC"    , m_handleRecChargedToMC);
+  declareOutput("recNeutralToMC"    , m_handleRecNeutralToMC);
+  declareOutput("recPhotonsToMC"    , m_handleRecPhotonsToMC);
+  declareOutput("recJetsToMC"       , m_handleRecJetsToMC);
+
+  // All variables truly initialized in the initialize() method (for pedantic C++ reasons also in a constructor)
+  m_stablePartOutArray = nullptr;
```

Comment:
The member initialiser list is above this point in the code :-).. I.e. where you initialise a lot of the other members.